### ISSUE TITLE
[WinPhone] does not support locales

### ISSUE THREAD

**Issue by hardeep** [phonegap/build#1]
Windows Phone defaults to the locale en_US, however users are not able to submit their apps as the correct locale is added to the project.

For example, Italian apps are not accepted to the market place as they are labelled en_US.

Parent Topic:
http://community.phonegap.com/nitobi/topics/_phonegap_build_introducting_windows_phone_7_builds?utm_content=topic_link&utm_medium=email&utm_source=reply_notification

**Comment by andreszs** [comment]
@goya What about version number? The external version number from the XML is completely ignored by the Windows Dev Center. Am I the first one to upload a XAP file from PhoneGap to Windows Dev Center, or what?

**Comment by goya** [comment]
Please read this thread as it explains how we normalize the version in config.xml to a valid version string for windows phone

http://community.phonegap.com/nitobi/topics/version-value-is-not-correct-in-windows-phone-8s-wmappmanifest-xml

**Comment by khaanYoufoot** [comment]
I have added that config-file element in my config.xml, i am using cordova cli, and the generated file still does not contain the language I want inside its WMAppManifest.xml

**Comment by wildabeast** [comment]
@khaanYoufoot the `<gap:config-file>` element is only supported by PhoneGap Build, it won't have any effect on cordova cli builds.

**Comment by khaanYoufoot** [comment]
@wildbeast < gap:config-file> is not supported by cordova CLI true, but it does affect i. It breaks the cordova CLI pp after splash screen

**Comment by ivanov84** [comment]
This work as plugin:

<platform name="windows">
        <config-file target="package.appxmanifest" parent="/Package/Resources" overwrite="true">
		<Resource Language="EN-GB" />
	</config-file>
</platform>

https://github.com/cranberrygame/cordova-plugin-supportedlanguages-en

**Comment by Firzenizer** [comment]
I am also having issues with Finnish language. Defaulting to en-US all the time. I tried the config element suggested by @goya with no luck. I am using phonegap build version 5.4.1.

I also noticed documentation has no mention of windows phone under modifying manifests:
http://docs.phonegap.com/phonegap-build/configuring/config-file-element/

What can I do? Except manually modify language every time.
